### DISCUSSION TITLE
Will webview2 allow direct WinRT API like webview does in UWP apps?

### DISCUSSION THREAD

**Discussion by DexterWoo** [microsoft/WindowsAppSDK#762] [Q&A]
We have this App that's using Webview Control and loading websites into it.
UWP Apps allows us to configure full WinRT API access from Website with Content URIs settings.

can Webview2 support the same feature?

**Comment by david-risney** [comment]
Great question! It likely won't work exactly the same way it did with the edgehtml WebView from WinUI 2. But we are working on adding WinRT support via CoreWebView2.AddHostObjectToScript which should allow you to accomplish the same things. If you have additional CoreWebView2 specific questions feel free to ask or file feature requests on [the WebView2Feedback project](https://github.com/MicrosoftEdge/WebView2Feedback/issues/). Thanks!
